pkg/operation/botanist/component/nginxingressshoot/nginxingress.go

```diff
@@ -156,17 +154,16 @@ func (n *nginxIngress) computeResourcesData() (map[string][]byte, error) {
 		ObjectMeta: metav1.ObjectMeta{
 			Name: configMapName,
 			Labels: map[string]string{
-				v1beta1constants.LabelApp: labelAppValue,
-				labelKeyRelease:           labelValueAddons,
-				labelKeyComponent:         labelValueController,
+				v1beta1constants.LabelApp:             labelAppValue,
+				labelKeyRelease:                       labelValueAddons,
+				labelKeyComponent:                     labelValueController,
+				references.LabelKeyGarbageCollectable: references.LabelValueGarbageCollectable,
```

Comment:
Add a comment why we don't use `MakeUnique` function here. Also why do we need this label when this config map is static? Also, I guess we should add a checksum now to the pod template?

Reply:
> Add a comment why we don't use MakeUnique function here

Okay.

> Also, I guess we should add a checksum now to the pod template?

Ah, yes.

> Also why do we need this label when this config map is static

We still need it to be garbage collected when we disabled the addon, right?

Reply:
> We still need it to be garbage collected when we disable the addon, right?

Garbage collection is only for unique resources - when the addon is disabled, the `ManagedResource` controller will delete all the "static" resources as usual.